### PR TITLE
Gradle8

### PR DESCRIPTION
**Pull request by rusefillc**

### CODE REVIEW

**Comment by rusefillc** [comment]
@mck1117 rebased fewer files are touched by the PR

File: java_console/ui/build.gradle [hunk 1]
```diff
@@ -21,9 +21,13 @@ dependencies {
 }
 
 shadowJar {
+    dependsOn(':enum_to_string:shadowJar')
+    dependsOn(':config_definition:shadowJar')
     archivesBaseName = 'fome_console'
-    destinationDir = file("$rootDir/../java_console_binary")
-    archiveClassifier = ''
+    base {
+        archivesBaseName = "${rootDir}/../java_console_binary/rusefi_console"
```
Comment (mck1117) [inline java_console/ui/build.gradle:28]: wrong binary name

Comment (mck1117) [reply java_console/ui/build.gradle:28]: also, does this not get specified on the previous line?

Comment (rusefillc) [reply java_console/ui/build.gradle]: @mck1117 try now. I guess the point is to specify destinationDir in a new way

**Pull request merged**